modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/utils/SSH.scala

```diff
@@ -53,11 +53,11 @@ object SSH {
     }
 
   /** Convert pure tunnel configuration to configuration with actual key and passphrase */
-  def getIdentity[F[_]: Monad: AWS](tunnelConfig: TunnelConfig): F[Identity] =
+  def getIdentity[F[_]: Monad: Sync](tunnelConfig: TunnelConfig): F[Identity] =
     tunnelConfig
       .bastion
       .key
-      .map(_.ec2ParameterStore.parameterName).traverse(AWS[F].getEc2Property)
+      .map(_.ec2ParameterStore.parameterName).traverse(AWS.getEc2Property[F])
```

Comment:
Similar implicit dependency between SSH and AWS. I think it should be validated in config.

Reply:
https://github.com/snowplow/snowplow-rdb-loader/pull/1024#discussion_r987793338